### PR TITLE
Comic fixes and additions

### PR DESCRIPTION
**Pull request by frater**
Fixed the URL for Cyanide & Happiness that prevented it being used via indexing.
Added several WebToon Comics.

### CODE REVIEW

File: dosagelib/plugins/c.py [hunk 1]
```diff
@@ -453,10 +453,10 @@ def namer(self, imageUrl, pageUrl):
 
 
 class CyanideAndHappiness(_BasicScraper):
-    url = 'http://www.explosm.net/'
+    url = 'http://explosm.net/comics/'
```
Comment (TobiX) [inline dosagelib/plugins/c.py:456]:
```suggestion
    url = 'https://explosm.net/comics/'
```
Seems to redirect, so maybe use HTTPS directly?

File: dosagelib/plugins/webtoons.py [hunk 2]
```diff
@@ -166,6 +167,7 @@ def getmodules(cls):  # noqa: Allowed to be long
             cls('FourLeaf', 'fantasy/four-leaf', 1454),
             cls('FreakingRomance', 'romance/freaking-romance', 1467),
             cls('FridayForbiddenTales', 'thriller/friday', 388),
+            cls('GameMasters', 'challenge/game-masters',237252),
```
Comment (TobiX) [inline dosagelib/plugins/webtoons.py:170]:
```suggestion
            cls('GameMasters', 'challenge/game-masters', 237252),
```
No new warnings, please

**Review by TobiX — COMMENTED** [review]
Looks good, just some minor nits!

File: dosagelib/plugins/webtoons.py (hunk 2 above)
Comment (frater) [reply dosagelib/plugins/webtoons.py:170]: ah, that's what the whitespace warning was about, I was wondering.

File: dosagelib/plugins/c.py (hunk 1 above)
Comment (frater) [reply dosagelib/plugins/c.py]: good idea, I missed that.

**Comment by TobiX** [comment]
Thanks!

**Pull request merged**